net/torsocks/Portfile

```diff
@@ -2,6 +2,11 @@
 
 PortSystem        1.0
 PortGroup           gitlab 1.0
+PortGroup           legacysupport 1.1
+PortGroup           openssl 1.0
```

Comment:
Are you sure you need to add the openssl portgroup? The port doesn't appear to link with openssl. I find no occurrences of "crypto" or "ssl" in the source code (except in the word "expressly" in the license text).

If you do need to add it, then you also need to increase the revision, since adding the portgroup causes a library dependency on openssl to be added to the port, which means it will now use MacPorts openssl instead of macOS openssl.

Reply:
No, sorry, this is a silly copy-paste mistake. I will remove it.